simplesat/constraints/parser.py

```diff
@@ -10,16 +10,20 @@
 from simplesat.errors import SolverException
 
 
-_VERSION_R = "[^=><!,\s;^][^,\s;]+"
-_EQUAL_R = "=="
-_GEQ_R = ">="
+# NOTE: re.Scanner seems to ignore (?i), so we have to write case-insensitivity
+# into the regex manually. We must also permit a leading underscore because of
+# names like `_distribute_remove`
```

Comment:
Can we reference the PEP defining this regex, and mention we remove `-` as this breaks too many things otherwise ?